lib/child_process.js

```diff
@@ -192,7 +197,7 @@ exports.execFile = function(file /*, args, options, callback*/) {
   }
 
   if (!callback && pos < arguments.length && arguments[pos] != null) {
-    throw new TypeError('Incorrect value of args option');
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'args', arguments);
```

Comment:
IIUC the error message would be `The value "['file', [], '']" is invalid for option "args"`  if we do `execFile('file', [], '')`, but then those are all the arguments, not `args` (which is `[]` in this case)